docs/cloud/visualize/dashboards.md

```diff
@@ -9,22 +9,22 @@ Click on the **Dashboards** tab in any War Room to get started.
 
 From the Dashboards tab, click on the **+** button.
 
-![Add or manage
-dashboards](https://user-images.githubusercontent.com/1153921/108529360-a2145d00-7291-11eb-814b-2ea3303beb64.png)
+<img width="98" alt=" Green plus button " src="https://github.com/netdata/netdata/assets/73346910/511e2b38-e751-4a88-bc7d-bcd49764b7f6"/>
+
 
 In the modal, give your new dashboard a name, and click **+ Add**.
 
-Click the **Add Chart** button to add your first chart card. From the dropdown, select either *All Nodes** or a specific
+- The **Add Chart** button on the top right of the interface adds your first chart card. From the dropdown, select either **All Nodes** or a specific
 node. If you select **All Nodes**, you will add a [composite chart](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/overview.md) to
-your new dashboard. Next, select the context. You'll see a preview of the chart before you finish adding it.
+your new dashboard. Next, select the context. You'll see a preview of the chart before you finish adding it. In this modal you can also [interact with the chart](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/interact-new-charts.md).
```

Comment:
I would say even more than just interact with the chart, you configure it before adding it to the dashboard:
* define group by
* selected nodes
* select instances
* select labels
* etc.

worth mentioning that after adding users can also change this settings and by hitting the save button they will be stored on the custom dashboard definition

Reply:
@hugovalente-pm check again, I added all the options and linked to the proper files for more info